src/transformers/models/mamba/modeling_mamba.py

```diff
@@ -50,7 +50,13 @@
     causal_conv1d_update, causal_conv1d_fn = None, None
 
 is_fast_path_available = all(
-    (selective_state_update, selective_scan_fn, causal_conv1d_fn, causal_conv1d_update, mamba_inner_fn)
+    (
+        selective_state_update,
+        selective_scan_fn,
+        causal_conv1d_fn,
+        causal_conv1d_update,
+        mamba_inner_fn,
+    )
 )
```

Comment:
this is unrelated and is styling, should be reverted!

Reply:
Thanks, I've updated the styling 👌